Extensions/PlatformBehavior/JsExtension.cpp

```diff
@@ -132,10 +132,12 @@ class PlatformBehaviorJsExtension : public gd::PlatformExtension {
           "simulateDownKey");
       autActions["PlatformBehavior::SimulateLadderKey"].SetFunctionName(
           "simulateLadderKey");
+      autActions["PlatformBehavior::SimulateReleaseLadderKey"].SetFunctionName(
+          "simulateReleaseLadderKey");
       autActions["PlatformBehavior::SimulateJumpKey"].SetFunctionName(
           "simulateJumpKey");
-      autActions["PlatformBehavior::SimulateReleaseKey"].SetFunctionName(
-          "simulateReleaseKey");
+      autActions["PlatformBehavior::SimulateReleasePlatformKey"].SetFunctionName(
```

Comment:
Unfortunately you must also keep a line for the deprecated `SimulateReleaseKey` (Because when duplicating the action, the code generation information is not yet filled).